src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
I saw that, but I'm not sure if we need `selectTranslation`. A translation is either created or updated if it already exists. Shouldn't that be handled in here https://github.com/zooniverse/pandora/blob/926ad562147496ff6ac195b27ae5f14c91c71801/src/ducks/resource.js#L129? I.e. We can have two actions instead of three